### PR TITLE
[circle-part-eval] Introduce Add_ part files

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce Add_Sqrt and Add_Sub partition files to test
multiple inputs or outputs of partitioning.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #6230, from draft #6231

**Comment by seanshpark** [comment]
https://github.com/Samsung/ONE/issues/6230#issuecomment-801680531 shows one of the model for the partition.

File: compiler/circle-part-eval/parts/Part_Add_Sqrt_000.part [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[partition]
+backends=cpu,acl_cl
+default=cpu
+comply=opcode
+
+[OPCODE]
+SQRT=acl_cl
```
Comment (mhs4670go) [inline compiler/circle-part-eval/parts/Part_Add_Sqrt_000.part:7]: There's no `Add`. Is it right?

Comment (seanshpark) [reply compiler/circle-part-eval/parts/Part_Add_Sqrt_000.part:7]: `ADD`, as not assigned explicity here in `OPCODE` section, then it's default: `cpu` is assigned.

Comment (seanshpark) [reply compiler/circle-part-eval/parts/Part_Add_Sqrt_000.part:7]: Or if my answer is not what you asked for, can you elaborate your question?

Comment (mhs4670go) [reply compiler/circle-part-eval/parts/Part_Add_Sqrt_000.part:7]: > ADD, as not assigned explicity here in OPCODE section, then it's default: cpu is assigned.

That exactly explains:)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**